REP can be migrated after forking period (#74)
AugurProject/whitepaper: v2/english/main.tex

During the forking period, holders of REP tokens in the parent universe may migrate their tokens to a child universe of their choice.  This choice should be considered carefully, because migration is one-way; it cannot be reversed.  Tokens cannot be sent from one sibling universe to another.  \textit{Migration is a permanent commitment of REP tokens to a particular outcome of the forking market.}  REP tokens that migrate to different child universes ought to be considered entirely separate tokens, and service providers like wallets and exchanges ought to list them as such.

> Any REP tokens that have not been migrated out of the parent universe when 60 days have passed since a fork occurred will be permanently locked in the parent universe. Such tokens are expected to lose all value, so it is of paramount importance that REP holders migrate their tokens anytime a fork happens.

Reviewer: ```suggestion
Any REP tokens that have not been migrated out of the parent universe 60 days after the fork started will be permanently locked in the parent universe. Such tokens are expected to lose all value, so it is of paramount importance that REP holders migrate their tokens anytime a fork happens.
```
